### ISSUE TITLE
Discord using UNSAFE_componentWillMount

### ISSUE THREAD

**Issue by rauenzi** [samogot/betterdiscord-plugins#36]
Discord overrides that function that we patched in the update *specifically* for the message component, I have not seen it anywhere else. Oddly enough they don't use `componentWillMount` on that object. What I have done for now is add this:

```js
monkeyPatch(React.Component.prototype, 'componentWillMount', {
	displayName: 'ReactComponent',
	instead: ({thisObject}) => {
		put(thisObject.constructor);
	}
});
```

right after the patch of `UNSAFE_componentWillMount` in the lib, this works for now and hopefully Discord won't keep targeting this plugin.

**Comment by samogot** [comment]
It looks like devs specially break Message patching

**Comment by samogot** [comment]
I'll release new version in evening if I won't forget

**Comment by deNoor** [comment]
You forgot :'(

**Comment by intrnl** [comment]
People are busy, not everyone has the time.

**Comment by samogot** [comment]
Fixed with lib v1.11
